sdk/core/core-rest-pipeline/src/util/stream.browser.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BlobLike, FileLike } from "../interfaces";
+
+export function isNodeReadableStream(x: unknown): x is NodeJS.ReadableStream {
```

Comment:
Some of those helpers are duplicated, is there a reason to?

Reply:
No particular reason. I have moved them to a separate file.